CONTRIBUTING.md

```diff
@@ -247,17 +249,17 @@ You can test Your Installation with the following steps.  These steps assume you
 ## Local testing tools
 FIBO developers are accustomed to using desktop tools like Protégé, TopBraid Composer, VOM, and MagicDraw/CCM.  These tools include a variety of tests that experienced users rely on to determine the correctness of their models. In order to satisfy themselves that the ontologies are correct, they need to be able to test the same configuration of ontologies that will be published (i.e., Production and Development, see above).  
```

Comment:
Not sure what are the "variety of tests" that are being referred to. 
Should also tell people where to get the Pellet plugin for Protege.